src/main/java/org/primefaces/model/DefaultScheduleEvent.java

```diff
@@ -183,8 +189,21 @@ public void setOverlapAllowed(boolean overlapAllowed) {
         this.overlapAllowed = overlapAllowed;
     }
 
+    /**
+     * Shortcut for calling both {@link #setDraggable(Boolean)} and
+     * {@link #setResizable(Boolean)}.
+     */
     public void setEditable(boolean editable) {
-        this.editable = editable;
+        this.draggable = editable;
+        this.resizable = editable;
+    }
+
+    public void setDraggable(Boolean draggable) {
+        this.draggable = draggable;
```

Comment:
Should these be lower "b" booleans?

Reply:
I am actually using Boolean objects on purpose here, see the docs on DefaultScheduleEvent.Builder#draggable: There are actually three possible scenarios: (a) draggable = false, which makes the event not draggable,overriding the default that was set on the Schedule component globally, (b) draggable = true, which makes the event always draggable, and (c) draggable = null (the default), which uses the default that was set on the schedule component.

I added some JavaDocs to DefaultScheduleEvent.Builder#draggable, but I guess I could add docs to this method, too.

Reply:
Cool as long as it was on purpose just mark this issue resolved.